### PR TITLE
Add round one of speakers

### PR DESCRIPTION
**Pull request by jessleenyc**

### CODE REVIEW

**Review by joshpuetz — CHANGES_REQUESTED** [review]
CodeLand_Conference_2020

White background is making text unreadable on Safari (both desktop and iPhone). Looks good on Chrome.

**Comment by vaidehijoshi** [comment]
I'm seeing the same issue as @joshpuetz; I suspect that Safari is inheriting a background color (?) for those speaker elements and needs an explicit background color set?

File: source/javascripts/main.js [hunk 1]
```diff
@@ -54,4 +54,22 @@ $(document).ready(function(){
...
+  // function closeModal() {
+  //   $(".mask").hide();
+  //   $(".speaker-item--details").hide();
+  //   $(".speaker-item.speaker-item--details").removeClass("speaker-item--details");
+  //   $(".speaker-item").show();
+  // }
+
+  // $(".mask").click(function(e) {
+  //   closeModal();
+  // })
```
Comment (vaidehijoshi) [inline source/javascripts/main.js:74]: Is this something we want to keep around for later?

File: source/layouts/_distributed_program.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          </div>
+          <div class="speaker-title"><strong><%= speaker.full_name %></strong>, <%= speaker.job_title %> <% if speaker.company %> at <%= speaker.company %> <% end %></div>
+        </div>
+      </div>
+      <div class="speaker-item--details">
+        <img src="<%= speaker.image_url %>">
+        <div class="speaker-title"><strong><%= speaker.full_name %></strong>, <%= speaker.job_title %> at <%= speaker.company %></div>
+        <%= speaker.bio %>
+      </div>
+      <% end %>
```
Comment (vaidehijoshi) [inline source/layouts/_distributed_program.erb]:
```suggestion
    <% end %>
```
I think this needs to be outdented

File: source/javascripts/main.js (hunk 1 above)
Comment (jessleenyc) [reply source/javascripts/main.js:74]: eh, i can probably remove it!

**Review by joshpuetz — APPROVED** [review]
Looks great! 👍

**Review by vaidehijoshi — APPROVED** [review]

**Pull request merged**